Makefile

```diff
@@ -46,14 +46,14 @@ test: build
 update-labels:
 	./hack/labels/update.sh
 
-install-metrics-binaries:
-	if ! command -V golangci-lint; then curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b ${GOPATH}/bin ${GOLANGCI_LINT_VERSION} ; fi
+check-linter-binaries:
+	if ! command -V golangci-lint; then echo "golangci-lint binary required to be installed locally" && exit 1; fi
```

Comment:
nit: move the check into `./hack/lint.sh` and remove the make target `check-linter-binaries`